wgsl/README.md

```diff
@@ -4,16 +4,41 @@
 
 The specification is written using [Bikeshed](https://tabatkins.github.io/bikeshed).
 
-If you have bikeshed installed locally, you can generate the specification with:
+If you have `bikeshed` installed locally, you can generate the specification with:
 
 ```
-make
+make index.html
 ```
 
-This simply runs bikeshed on the `index.bs` file.
+This runs bikeshed on the [`index.bs`](index.bs) file.
 
 Otherwise, you can use the bikeshed Web API:
 
 ```
 make online
 ```
+
+## Checking that WGSL code samples parse correctly
+
+To check that the WGSL source code samples in the specification parse correctly, run:
+
+```
+make grammar/grammar.js
+```
+
+This uses:
+* [node.js](https://nodejs.org/)
+* [npm](https://www.npmjs.com/)
+* [Python 3](https://www.python.org/)
```

Comment:
I think it might be good to note that Node version must be greater than 16 and Python version must be greater than 3.8 to ensure our usage of some imports don't create headache for people (especially regex which did change a bit over .x versions of Python).

Reply:
Done